src/App4.js

```diff
@@ -0,0 +1,31 @@
+React.useEffect(() => {
+    if(age && name == '1') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/shekhar-rudder/code-code/pullRequest?prid=7912542)
```suggestion
    if(age && name == "1") {
```